Src/Fido2/IMetadataService.cs

```diff
@@ -4,17 +4,14 @@
 
 namespace Fido2NetLib
 {
-    /// <summary>
-    /// The FIDO2 metadata service.
-    /// </summary>
     public interface IMetadataService
     {
         /// <summary>
-        /// Gets the metadata payload entry by a guid.
+        /// Gets the metadata payload entry by a guid asyncronously
         /// </summary>
         /// <param name="aaguid">The Authenticator Attestation GUID.</param>
         /// <returns>Returns the entry; Otherwise <c>null</c>.</returns>
-        MetadataBLOBPayloadEntry? GetEntry(Guid aaguid);
+        Task<MetadataBLOBPayloadEntry?> GetEntryAsync(Guid aaguid, CancellationToken cancellationToken = default);
 
         /// <summary>
         /// Gets a value indicating whether the internal access token is valid.

```

Comment:
Is this comment really correct? I think it is wrong. 
I think ConformanceTesting() is used to inform the library if we're running in a conformance testing environment where "special rules" apply? @aseigler am I misstaken?